tests/tck/features/optimizer/PrunePropertiesRule.feature

```diff
@@ -656,68 +648,60 @@ Feature: Prune Properties rule
       | 33                | "男"         |
     When executing query:
       """
-      match (v:player) return id(v),properties(v).name,v.player.age limit 2;
-      """
-    Then the result should be, in any order, with relax comparison:
-      | id(v)               | properties(v).name  | v.player.age |
-      | "Amar'e Stoudemire" | "Amar'e Stoudemire" | 36           |
-      | "Carmelo Anthony"   | "Carmelo Anthony"   | 34           |
-    When executing query:
-      """
-      match (v) return id(v),properties(v).name,v.player.age limit 2;
+      match (v:player) return id(v) AS id, properties(v).name AS name, v.player.age AS age order by name limit 2;
       """
     Then the result should be, in any order, with relax comparison:
-      | id(v)               | properties(v).name  | v.player.age |
-      | "Amar'e Stoudemire" | "Amar'e Stoudemire" | 36           |
-      | "Carmelo Anthony"   | "Carmelo Anthony"   | 34           |
+      | id                  | name                | age |
+      | "Amar'e Stoudemire" | "Amar'e Stoudemire" | 36  |
+      | "Aron Baynes"       | "Aron Baynes"       | 32  |
     When executing query:
       """
-      match ()-[e]->() return properties(e).start_year limit 2;
+      match (v)-[]->(b:player) return id(v) AS id, properties(v).name AS name, v.player.age AS age order by name limit 2;
       """
     Then the result should be, in any order, with relax comparison:
```

Comment:
As discussed, since added order by, the result should be verified in fix order now, not in any order

Reply:
fixed